yarn.lock

```diff
@@ -11562,14 +11562,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"lodash@npm:3.x":
-  version: 3.10.1
-  resolution: "lodash@npm:3.10.1"
-  checksum: 10/3fa24526ced39885889d5609239d14c4145b3736f103fcdde247524203bfeb38e641653df7bdcc172b316a9775ca32787dda09430872ce906f017a699c92fad8
-  languageName: node
-  linkType: hard
-
-"lodash@npm:>=3.5 <5, lodash@npm:^4.17.13, lodash@npm:^4.17.14, lodash@npm:^4.17.15, lodash@npm:^4.17.19, lodash@npm:^4.17.21, lodash@npm:^4.17.4, lodash@npm:~4.17.12":
+"lodash@npm:~4.17.12":
```

Comment:
I noticed that some of the packages require versions higher than 4.17.12, so I'm curious why you didn't choose, say 4.7.21 (since that is what it resolved to anyway?

Reply:
Fixed with 4.7.21